src/Tools/dotnet-trace/CommandLine/Commands/ConvertCommand.cs

```diff
@@ -12,21 +14,22 @@ internal static class ConvertCommandHandler
     {
         public static int ConvertFile(IConsole console, FileInfo inputFilename, TraceFileFormat format, FileInfo output)
         {
-            if ((int)format <= 0)
+            if (!Enum.IsDefined(format))
             {
-                Console.Error.WriteLine("--format is required.");
+                string options = string.Join(", ", Enum.GetNames(typeof(TraceFileFormat)).Where(x => x != TraceFileFormat.NetTrace.ToString()));
+                console.Error.WriteLine($"Please specify a valid option for the --format. Valid options are: {options}.");
                 return ErrorCodes.ArgumentError;
             }
 
             if (format == TraceFileFormat.NetTrace)
             {
-                Console.Error.WriteLine("Cannot convert a nettrace file to nettrace format.");
+                console.Error.WriteLine("Cannot convert a nettrace file to nettrace format.");
                 return ErrorCodes.ArgumentError;
```

Comment:
Does it make sense to throw an error? If there is still utility to having a NetTrace option (is there?), then would it be better to just log to the user that the file being converted is already a NetTrace (which it doesn't seem like we have any checks) and return success?

Reply:
I do think it would be nice if this case wasn't an error at all, but in order to do that we need to copy the input file to the output file path.